codec/src/test/java/io/netty/handler/codec/compression/AbstractIntegrationTest.java

```diff
@@ -155,7 +155,7 @@ protected void testIdentity(final byte[] data) {
         assertThat(compressed, is(notNullValue()));
 
         decoder.writeInbound(compressed.retain());
-        assertFalse(compressed.isReadable());
+        //assertFalse(compressed.isReadable());
```

Comment:
revert ?

Reply:
This is commented because the internal behaviour changed slightly - the input buffer might not always have its `readerIndex` advanced if it gets released (since the pipeline is the "owning" reader, it didn't make sense to me to adjust the `readerIndex` immediately prior to releasing). But this can of course be discussed/changed if decided...